Celeste.Mod.mm/Mod/Core/CoreModuleSettings.cs

```diff
@@ -222,17 +222,16 @@ public class CoreModuleSettings : EverestModuleSettings {
             }
         }
 
-        private bool disableDiscordRichPresence = false;
-
         public bool DisableDiscordRichPresence {
-            get => disableDiscordRichPresence;
+            get => !Everest.Discord.Initialized;
             set {
+                if (value != Everest.Discord.Initialized)
+                    return;
```

Comment:
This check should ideally be moved to `Everest.Discord` as there *could* be f.e. a mod calling `.Initialize()` without first checking `.Initialized` - it is fine as is though.